pkg/network/network.go

```diff
@@ -58,8 +61,26 @@ const (
 	// DefaultDomainTemplate is the default golang template to use when
 	// constructing the Knative Route's Domain(host)
 	DefaultDomainTemplate = "{{.Name}}.{{.Namespace}}.{{.Domain}}"
+
+	// TLSModeKey is the name of the configuration entry
+	// that specifies the TLS Mode of Knative.
+	TLSModeKey = "tlsMode"
+
+	// DefaultTLSMode is the default TLS mode of Knative. It is `MANUAL` by default.
+	DefaultTLSMode = Manual
+)
+
+const (
+	// Manual represents the `Manual` TLS MODE.
+	Manual = "Manual"
+	// Auto represents the `Auto` TLS MODE.
+	Auto = "Auto"
+	// Force represents the `Force` TLS MODE.
+	Force = "Force"
```

Comment:
Do these need to be public?  If so, should we qualify them?  `network.Manual` is not very clear, but `network.ManualTLS` is...